docs/docs/getting-started.mdx

```diff
@@ -911,6 +934,8 @@ See its readme on how to configure it.
 
 [build-system-vite]: #vite
 
+[bun]: https://bun.sh/
```

Comment:
```suggestion
[bun]: https://bun.sh

[javascript-engines-bun]: #bun
```

- slash unneeded, rest of docs don’t use it either
- also adding link for bun section